aten/src/ATen/core/jit_type.h

```diff
@@ -1449,6 +1449,27 @@ struct CAFFE2_API ClassType : public NamedType {
     return attributeTypes_[pos];
   }
 
+  TypePtr getAttribute(const std::string& name) const {
```

Comment:
Let's avoid this code duplication by having `getAttribute` call `findAttribute` internally.